fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexTest.java

```diff
@@ -1337,6 +1341,158 @@ class GroupSpec {
         }
     }
 
+    private LuceneScanQuery buildLuceneScanQuery(Index index,
+                                                 boolean isSynthetic,
+                                                 Comparisons.Type comparisonType,
+                                                 SortType sortType,
+                                                 long predicateComparand,
+                                                 String luceneSearch) {
+        final Sort sort;
+        if (sortType == SortType.UNSORTED) {
+            sort = null;
+        } else {
+            sort = new Sort(new SortField(isSynthetic ? "complex_timestamp" : "timestamp", SortField.Type.LONG, sortType == SortType.DESCENDING));
+        }
+
+        final List<LuceneQueryClause> luceneQueryClauses;
+
+        if (comparisonType == Type.NOT_EQUALS) {
```

Comment:
This is not actually used in the test.

Reply:
it is used by the other unit test that calls `buildLuceneQuery` in the other PR, from which this method was extracted.